manager/src/main/java/com/redhat/service/bridge/manager/ProcessorService.java

```diff
@@ -1,107 +1,26 @@
-import java.util.stream.Collectors;
-
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import javax.transaction.Transactional;
 
 import com.redhat.service.bridge.infra.models.dto.BridgeStatus;
 import com.redhat.service.bridge.infra.models.dto.ProcessorDTO;
 import com.redhat.service.bridge.manager.api.models.requests.ProcessorRequest;
-import com.redhat.service.bridge.manager.dao.ProcessorDAO;
-import com.redhat.service.bridge.manager.exceptions.AlreadyExistingItemException;
-import com.redhat.service.bridge.manager.exceptions.BridgeLifecycleException;
-import com.redhat.service.bridge.manager.exceptions.ItemNotFoundException;
-import com.redhat.service.bridge.manager.models.Action;
-import com.redhat.service.bridge.manager.models.Bridge;
-import com.redhat.service.bridge.manager.models.Filter;
 import com.redhat.service.bridge.manager.models.ListResult;
 import com.redhat.service.bridge.manager.models.Processor;
 
-@Transactional
-@ApplicationScoped
-public class ProcessorService {
-
-    @Inject
-    ProcessorDAO processorDAO;
-
-    @Inject
-    BridgesService bridgesService;
-
-    public Processor getProcessor(String processorId, String bridgeId, String customerId) {
-
-        Bridge bridge = bridgesService.getBridge(bridgeId, customerId);
-        Processor processor = processorDAO.findByIdBridgeIdAndCustomerId(processorId, bridge.getId(), bridge.getCustomerId());
-        if (processor == null) {
-            throw new ItemNotFoundException(String.format("Processor with id '%s' does not exist on Bridge '%s' for customer '%s'", processorId, bridgeId, customerId));
-        }
-
-        return processor;
-    }
-
-    public Processor createProcessor(String bridgeId, String customerId, ProcessorRequest processorRequest) {
-        Bridge bridge = getAvailableBridge(bridgeId, customerId);
-        if (processorDAO.findByBridgeIdAndName(bridgeId, processorRequest.getName()) != null) {
-            throw new AlreadyExistingItemException("Processor with name '" + processorRequest.getName() + "' already exists for bridge with id '" + bridgeId + "' for customer '" + customerId + "'");
-        }
-
-        final Processor p = new Processor();
-
-        Set<Filter> filters = null;
-        if (processorRequest.getFilters() != null) {
-            filters = processorRequest.getFilters().stream().map(x -> new Filter(x.getKey(), x.getType(), x.getValueAsString(), p)).collect(Collectors.toSet());
-        }
-
-        p.setName(processorRequest.getName());
-        p.setSubmittedAt(ZonedDateTime.now());
-        p.setStatus(BridgeStatus.REQUESTED);
-        p.setBridge(bridge);
-        p.setFilters(filters);
-
-        Action a = new Action();
-        a.setName(processorRequest.getAction().getName());
-        a.setParameters(processorRequest.getAction().getParameters());
-        a.setType(processorRequest.getAction().getType());
-        p.setAction(a);
+public interface ProcessorService {
```

Comment:
Moved to an interface... unfortunately github shows a complete rewrite, but this is not the case

Reply:
Is this just a refactor or `ProcessorServiceImpl` contains also some changes?